app/views/simple_calendar/_week_calendar.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="simple-calendar">
```

Comment:
Are this file and the _month_calendar and _calendar file automatically added by the gem? Looks like you aren't using all three of these files